bims/static/js/views/olmap_basemap.js

```diff
@@ -100,11 +108,22 @@ define(['backbone', 'underscore', 'jquery', 'ol', 'olMapboxStyle'], function (Ba
                 baseSourceLayers.push(bingMap);
             }
 
+            // OSM MAPSURFER ROADS - Make default
+            var mapSurfer = new ol.layer.Tile({
+                title: 'OSM Mapsurfer roads',
+                source: new ol.source.XYZ({
+                    attributions: ['OpenStreetMap contributors, ODbL, Imagery GIScience Research Group @ Heidelberg University'],
+                    url: 'http://korona.geog.uni-heidelberg.de/tiles/roads/x={x}&y={y}&z={z}'
```

Comment:
can you find the https version?

Reply:
Alright
Ill take note of that lets-encrypt effect.
Thanks